test/integration/CalldataExecute.t.sol

```diff
@@ -28,8 +28,126 @@ contract CalldataExecute is Test, Configs {
     }
 
     function testSimulateExec() public {
-        vm.warp(block.timestamp + 24 hours);
-        TemporalGovernor(addresses.getAddress("TEMPORAL_GOVERNOR"))
-            .executeProposal(execData);
```

Comment:
= We should consider testing a proposal calldata that includes actions on the base. We can then mock the VAA, similar to how we did for the hybrid proposal, and check that the proposal passes on base.